lua/entities/gmod_wire_expression2/core/serialization.lua

```diff
@@ -1,11 +1,11 @@
-E2Lib.RegisterExtension("glon", true)
+E2Lib.RegisterExtension("serialization", true)
```

Comment:
This might be problematic, since it turns stuff back on that people purposely disabled.
Maybe you can split this up to 3 registered extensions